scripts/sysman/temperature.yml

```diff
@@ -21,12 +21,18 @@ etors:
       desc: "The maximum temperature across all sensors in the GPU"
     - name: MEMORY
       desc: "The maximum temperature across all sensors in the local memory"
+    - name: GPU_BOARD
```

Comment:
since we have "MIN" on the other metric, this could be "GPU_BOARD_MAX" ?

Reply:
Agreed. The existing enum values didnt have this either - hence kept it to look uniform for all enum values.

Reply:
new enums need to go at the bottom of the list.

Reply:
would it be good to expose both front and back temps here? i.e, GPU_BOARD_FRONT, GPU_BOARD_BACK and equivalent _MIN enums)?

Reply:
There are 2 board temperature sensors. One of them will be MIN and the other MAX.

Reply:
Overall the PR lgtm. +1